go.mod

```diff
@@ -21,7 +21,7 @@ require (
 	github.com/spf13/viper v1.10.1
 	github.com/stretchr/testify v1.7.0
 	github.com/tangzero/inflector v1.0.0
-	go.mongodb.org/atlas v0.14.0
+	go.mongodb.org/atlas v0.14.1-0.20220104180936-6afea6036494
```

Comment:
Out of curiosity, what would be the difference between `v0.14.1-0.20220104180936-6afea6036494` and `v0.14.1`?

Reply:
if you go here [MongoDB Atlas Client Release v0.14.0](https://github.com/mongodb/go-client-mongodb-atlas/releases/tag/v0.14.0), you see that this release doesn’t have 4 commits from master
![Screenshot 2022-01-12 at 15 55 25](https://user-images.githubusercontent.com/5663078/149174878-1dfaa008-c6d6-46c4-8421-ba86e9081b7a.png). In particular, the one that we are using here is https://github.com/mongodb/go-client-mongodb-atlas/pull/271

Reply:
Thanks, so it's a kind of pre-release for `v0.14.1`. My comment is non-blocking btw.